features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1422,6 +1422,17 @@ CREATE TABLE IDN_FUNCTION_LIBRARY (
 )
 /
 
+CREATE TABLE IDN_FEDERATED_AUTH_SESSION_MAPPING (
+	IDP_SESSION_ID VARCHAR(255) NOT NULL,
+	SESSION_ID VARCHAR(255) NOT NULL,
+	IDP_NAME VARCHAR(255) NOT NULL,
+	AUTHENTICATOR_ID VARCHAR(255),
+	PROTOCOL_TYPE VARCHAR(255),
```

Comment:
Can these two be null?

Reply:
I'm not using the values of these two fields. Added these two for future work.